specification/batch/data-plane/Microsoft.Batch/2017-09-01.6.0/BatchService.json

```diff
@@ -12605,17 +12717,88 @@
           "$ref": "#/definitions/WindowsConfiguration",
           "title": "Windows operating system settings on the virtual machine.",
           "description": "This property must not be specified if the imageReference or osDisk property specifies a Linux OS image."
+        },
+        "dataDisks": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DataDisk"
+          },
+          "title": "The configuration for data disks attached to the Comptue Nodes in the pool.",
```

Comment:
capitalisation and typo (comptue -> compute)

Reply:
Fixed